src/lib/Components/Bidding/Elements/Typography.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import styled from "styled-components/native"
+import { color, fontSize, lineHeight, space, textAlign } from "styled-system"
+
+import { Fonts } from "../../../data/fonts"
+
+const Serif = styled.Text`
+  font-family: "${Fonts.GaramondRegular}";
+  ${color}
+  ${fontSize}
+  ${lineHeight}
+  ${space}
+  ${textAlign}
+`
+
+const SerifItalic = styled.Text`
+  font-family: "${Fonts.GaramondItalic}";
+  ${color}
+  ${fontSize}
+  ${lineHeight}
+  ${space}
+  ${textAlign}
+`
+
+export const Serif12 = props => <Serif fontSize={1} {...props} />
+export const Serif14 = props => <Serif fontSize={2} {...props} />
+export const Serif16 = props => <Serif fontSize={3} {...props} />
+export const Serif18 = props => <Serif fontSize={4} {...props} />
+
+export const SerifItalic10 = props => <SerifItalic fontSize={0} {...props} />
+export const SerifItalic12 = props => <SerifItalic fontSize={1} {...props} />
+export const SerifItalic14 = props => <SerifItalic fontSize={2} {...props} />
+export const SerifItalic16 = props => <SerifItalic fontSize={3} {...props} />
+export const SerifItalic18 = props => <SerifItalic fontSize={4} {...props} />
```

Comment:
We are only using some of these, but added them just to see what it's going to look like.

Reply:
I was still figuring out the best approach for our font system. But I appreciate the simplicity of just defining each value that way. Will share an alternative shortly in my reaction PR